packages/burpsuite/PKGBUILD

```diff
@@ -2,14 +2,15 @@
 # See COPYING for license details.
 
 pkgname=burpsuite
-pkgver=2021.5.3
+pkgver=2021.6.2
 pkgrel=1
 epoch=1
 groups=('blackarch' 'blackarch-webapp' 'blackarch-proxy' 'blackarch-scanner'
         'blackarch-fuzzer')
 pkgdesc='An integrated platform for attacking web applications (community edition) + SHELLING plugin.'
 arch=('any')
 url='https://portswigger.net/burp/releases/professional-community-2021-2-1?requestededition=community'
+url='https://portswigger.net/burp/releases/community/latest'
```

Comment:
Why do you add a `url` instead of updating the old one?

Please update `url` instead, as it is outdated anyways

Reply:
whoops, overlooked it by mistake